liquibase-standard/src/main/java/liquibase/statement/core/AddAutoIncrementStatement.java

```diff
@@ -28,15 +26,24 @@ public AddAutoIncrementStatement(
             BigInteger incrementBy,
             Boolean defaultOnNull,
             String generationType) {
-        this.catalogName = catalogName;
-        this.schemaName = schemaName;
-        this.tableName = tableName;
+        this.databaseTableIdentifier.setCatalogName(catalogName);
+        this.databaseTableIdentifier.setSchemaName(schemaName);
+        this.databaseTableIdentifier.setTableName(tableName);
         this.columnName = columnName;
         this.columnDataType = columnDataType;
         this.startWith = startWith;
         this.incrementBy = incrementBy;
         this.defaultOnNull = defaultOnNull;
         this.generationType = generationType;
     }
+    public String getCatalogName() {
```

Comment:
I think we don't need these getters here as this just a build-up statement class.